lib/MusicBrainz/Server/Data/Genre.pm

```diff
@@ -0,0 +1,81 @@
+package MusicBrainz::Server::Data::Genre;
+
+use Moose;
+use MusicBrainz::Server::Data::Utils qw(
+    hash_to_row
+    load_subobjects
+    merge_string_attributes
+    merge_table_attributes
+    order_by
```

Comment:
merge_string_attributes, merge_table_attributes, and order_by are unused in this file